### PR TITLE
Updated code to generate designer.cs file only if resx file name does not have locale

### PR DESCRIPTION
**Pull request by kirthik**
cc @troydai

### CODE REVIEW

**Comment by troydai** [comment]
Looks good to me. Let's hear from @pranavkm 
Consider I sign off this.

File: build/_k-standard-goals.shade [hunk 1]
```diff
@@ -192,7 +194,21 @@ default Quiet='${ false }'
    @{
      foreach (var file in Directory.EnumerateFiles(BASE_DIR, "*.resx", SearchOption.AllDirectories))
      {
-        UpdateResx(file);
+        var splitFileName = file.Split(new string[] { "." }, StringSplitOptions.None);
+
+        if (splitFileName.Length > 2)
+        {
+            var cultures = CultureInfo.GetCultures(CultureTypes.NeutralCultures | CultureTypes.InstalledWin32Cultures | CultureTypes.SpecificCultures);
```
Comment (pranavkm) [inline build/_k-standard-goals.shade]: Can we pull this outside of the the `foreach` loop?

File: build/_k-standard-goals.shade [hunk 2]
```diff
@@ -192,7 +194,21 @@ default Quiet='${ false }'
    @{
      foreach (var file in Directory.EnumerateFiles(BASE_DIR, "*.resx", SearchOption.AllDirectories))
      {
-        UpdateResx(file);
+        var splitFileName = file.Split(new string[] { "." }, StringSplitOptions.None);
```
Comment (pranavkm) [inline build/_k-standard-goals.shade]: Could you split the result of `Path.GetFileNameWithoutExtension` here?

**Comment by pranavkm** [comment]
Looks good otherwise. :shipit:

File: build/_k-standard-goals.shade (hunk 1 above)
Comment (kirthik) [inline build/_k-standard-goals.shade]: Will do.

File: build/_k-standard-goals.shade (hunk 2 above)
Comment (kirthik) [inline build/_k-standard-goals.shade]: Yes. Will change it.

**Pull request closed without merging**